src/FSharpx.Extras/ComputationExpressions/Monad.fs

```diff
@@ -232,12 +231,6 @@ module Option =
         | Some x -> x
         | None -> failwith m
 
-    /// Gets the value associated with the option or print to a string buffer and raise an exception with the given result. Helper printers must return strings.
-    let inline getOrFailF fmt =
-        function
-        | Some x -> x
-        | None -> failwithf fmt
-
```

Comment:
Rather than just removing this function, please add an `ObsoleteAttribute` to explain that it will be removed in a future release and which function should be used instead.

Reply:
the function has been introduced quite recently, and it's broken by design from the beginning and there's no substitute -- it just doesn't work